packit/upstream.py

```diff
@@ -875,6 +889,8 @@ def _fix_specfile_to_use_local_archive(self, archive: str) -> None:
         current_commit = self.upstream.local_project.commit_hexsha
         env = {
             "PACKIT_PROJECT_VERSION": self.current_git_describe_version,
+            # Spec file %release field which packit sets by default
+            "PACKIT_RPMSPEC_RELEASE": self.upstream.get_spec_release(),
```

Comment:
Using `PACKIT_PROJECT_RELEASE` would be consistent but a bit misleading. I like the `PACKIT_RPMSPEC_RELEASE`.